src/app/public/modules/data-manager/fixtures/data-manager.component.fixture.html

```diff
@@ -1,9 +1,17 @@
 <sky-data-manager>
   <sky-data-manager-toolbar>
+    <sky-data-manager-toolbar-primary-item>
+      <button
+        type="button"
+        class="sky-btn-primary"
```

Comment:
Not sure how much this matters since it's a test fixture, but in a real-world scenario you'd also need to specify the `sky-btn` CSS class.